packages/notifi-react-card/lib/components/subscription/defaults.css

```diff
@@ -149,6 +149,8 @@
   line-height: 14px;
   color: var(--notifi-secondary-font-color);
   text-align: center;
+  height: 400px;
+  line-height: 400px;
```

Comment:
I think we shouldn't use line-height to try and center the text.

Also, the alert description shouldn't take up a height of 400px, instead, the base card should have a default height defined and this component should grow to fill the  space.